core/client/fs/src/main/java/alluxio/client/file/ufs/UfsBaseFileSystem.java

```diff
@@ -468,7 +470,11 @@ private URIStatus transformStatus(UfsStatus ufsStatus, String ufsFullPath) {
       UfsFileStatus fileStatus = (UfsFileStatus) ufsStatus;
       info.setLength(fileStatus.getContentLength());
       info.setBlockSizeBytes(fileStatus.getBlockSize());
-    } else {
+      info.setUfsFingerprint(
+          Fingerprint.create(mUfs.get().getUnderFSType(), ufsStatus, fileStatus.getContentHash())
```

Comment:
If we can get the contentHash by `fileStatus.getContentHash()`, why do we need to pass the contentHash directly to this `Fingerprint.create( )` function?

Reply:
Do we also need to add the Fingerprint String to FileInfo in DoraPagedWorker##buildFileInfoFromUfsStatus()?